README.md

```
@@ -1,85 +1,119 @@
+
 # FreakyKit.Utils
 
-<div>
-   <a href="https://www.nuget.org/packages/FreakyKit.Utils"><img src="https://img.shields.io/nuget/v/FreakyKit.Utils?color=blue&logo=nuget"></a>
-   <a href="https://www.nuget.org/packages/FreakyKit.Utils"><img src="https://img.shields.io/nuget/dt/FreakyKit.Utils.svg"></a>
-   <a href="./LICENSE"><img src="https://img.shields.io/github/license/freakyali/FreakyKit.Utils"></a>
-   <a href="https://www.codefactor.io/repository/github/freakyali/FreakyKit.Utils"><img src="https://www.codefactor.io/repository/github/freakyali/FreakyKit.Utils/badge"></a>
-   <a href="https://app.fossa.com/projects/git%2Bgithub.com%2FFreakyAli%2FFreakyKit.Utils?ref=badge_shield" alt="FOSSA Status"><img src="https://app.fossa.com/api/projects/git%2Bgithub.com%2FFreakyAli%2FFreakyKit.Utils.svg?type=shield"/></a>
+**A lightweight, dependency-free collection of C# extension methods for .NET**
+
+<a href="https://www.nuget.org/packages/FreakyKit.Utils"><img src="https://img.shields.io/nuget/v/FreakyKit.Utils?color=blue&logo=nuget&style=for-the-badge"></a>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)